modules/vaos/app/services/vaos/user_service.rb

```diff
@@ -16,7 +16,7 @@ def session(user)
     def create_session(user)
       url = '/users/v2/session?processRules=true'
       token = VAOS::JWT.new(user).token
-      response = perform(:post, url, token, 'Content-Type' => 'text/plain', 'Referer' => 'https://api.va.gov')
+      response = perform(:post, url, token)
       if response&.body
         SessionStore.new(user_uuid: user.uuid, token: response.body).save
         response.body
```

Comment:
This returns either a `response.body` from the `if` or a `response`. Seems like you might want something more predictable in the event of an `else` since this is the output that goes into session creation.

I'd also rather see the storage of the session token happen closer to the cache check in the `session` method.

Reply:
I think from testing what we've seen is that anything not returning a response body will raise an exception which would break execution and not cache, but this is a good catch! There might be some edge case we have unaccounted for which could present difficult to debug scenarios with current handling (in the event we return a 2xx without response body).